### DISCUSSION TITLE
Support for Corsair iCUE H170i Elite Capellix XT

### DISCUSSION THREAD

**Discussion by sallymanda** [liquidctl/liquidctl#630] [Q&A]
Hey everyone, so happy this project exists, I'm building a new PC and it's certainly simplified my search for a CPU cooler 😊 Just wondering if I'll be able to control fan and pump speeds for a Corsair H170i? Hopefully as I can't seem to find a 420mm AIO that'll play well with Linux 😑

**Comment by tuxcrafter** [comment]
Did you find a good way to get the Corsair iCUE H170i CAPELLIX XT working under Linux?

**Reply by sallymanda** [reply]
Not a good way, but there is _a way_:
1. Setup Windows on a virtual machine
2. Passthrough the Corsair USB to the virtual machine
3. Install the Corsair software
4. Setup your lighting setup in the Corsair software, and itll persist between boots, but youll have to reopen the VM if you want to change it
